libraries/AP_Networking/AP_Networking.cpp

```diff
@@ -89,7 +89,7 @@ const AP_Param::GroupInfo AP_Networking::var_info[] = {
     // @Param: OPTIONS
     // @DisplayName: Networking options
     // @Description: Networking options
-    // @Bitmask: 0:EnablePPP Ethernet gateway
+    // @Bitmask: 0:EnablePPP Ethernet gateway,1:Enable AUTOIP (Link Local IPv4 Addressing)
```

Comment:
Instead of this approach, we could also make a parameter `NET_AUTOIP`, which would be able to be compiled out of the build if someone wants to save flash. 

Thoughts?

Reply:
No NET_AUTOIP, option is good according to the dev team.